sys/random/Kconfig

```diff
@@ -19,14 +19,20 @@ choice RANDOM_IMPLEMENTATION
     default MODULE_PRNG_HWRNG if HAS_PERIPH_HWRNG
     default MODULE_PRNG_TINYMT32
 
-config MODULE_PRNG_FORTUNA
+menuconfig MODULE_PRNG_FORTUNA
     bool "Fortuna"
     select MODULE_HASHES
-    select MODULE_XTIMER
     select MODULE_FORTUNA
     select MODULE_CRYPTO
     select MODULE_CRYPTO_AES_128
 
+if MODULE_PRNG_FORTUNA
+
+config MODULE_FORTUNA_RESEED
```

Comment:
I know it is not strictly related to the PR, but would you mind adding the `MODULE_ATOMIC_UTILS` to Kconfig? It seems it has no dependencies.